physionet-django/console/views.py

```diff
@@ -119,20 +119,18 @@ def handling_view(request, *args, **kwargs):
 # ------------------------- Views begin ------------------------- #
 
 
-@login_required
-@user_passes_test(is_admin, redirect_field_name='project_home')
+@user_passes_test(is_admin, login_url='/login/')
```

Comment:
This is incorrect - visiting `/console` yields a redirect loop.

Reply:
@kshalot can handle it.

Reply:
Create an issue on GH about this. The  `is_admin` flag is used to display the console, which is hacky but is a very simple solution that works. We decided to keep it for now.

Reply:
https://github.com/MIT-LCP/physionet-build/issues/1540